Plugins/NativeEngine/Source/NativeEngine.cpp

```diff
@@ -645,7 +645,7 @@ namespace Babylon
         const uint32_t byteOffset = info[2].As<Napi::Number>().Uint32Value();
         const uint32_t byteLength = info[3].As<Napi::Number>().Uint32Value();
 
-        vertexBuffer->Update(info.Env(), gsl::make_span(static_cast<uint8_t*>(bytes.Data()) + byteOffset, byteLength));
+        vertexBuffer->Update(info.Env(), gsl::make_span(static_cast<uint8_t*>(bytes.Data()), byteLength), byteOffset);
```

Comment:
I'm confused. This is updating a dynamic buffer, but the dynamic path isn't updated in VertexBuffer?

Reply:
I still don't get it. Static vertex buffers can be updated? I thought only dynamic vertex buffers can be updated. What am I missing?

Reply:
There 3 types of vertex buffer: GPU static, GPU dynamic and CPU. The last one is only available when used with Thin Instancing. In this case, datas are moved to GPU each frame : https://github.com/BabylonJS/BabylonNative/blob/1abe1ef7f52cd1330a57c42d178ddb03a36486bd/Plugins/NativeEngine/Source/VertexBuffer.cpp#L169

This PR covers this case where a subset of the CPU mem needs to be updated.